lib/engine/config/tile.rb

```diff
@@ -490,13 +490,17 @@ module Tile
         '1200' => 'city=revenue:10;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:4,b:_0;path=a:5,b:_0',
       }.freeze
 
+      BROWNSEPIA = {
```

Comment:
just call it sepia?

Reply:
No. It's a striped tile with both brown and sepia